core/tauri-runtime-wry/Cargo.toml

```diff
@@ -13,7 +13,7 @@ exclude = [ ".license_template", "CHANGELOG.md", "/target" ]
 readme = "README.md"
 
 [dependencies]
-wry = { version = "0.20", default-features = false, features = [ "file-drop", "protocol" ] }
+wry = { git = "https://github.com/tauri-apps/wry", branch = "auto-focus",default-features = false, features = [ "file-drop", "protocol" ] }
```

Comment:
Should this be changed to target the `dev` branch now that https://github.com/tauri-apps/wry/pull/676 has been merged?

Reply:
I was using git in this PR, just to make sure the changes passes Tauris CI. Once a new WRY version is published this will be reverted to point back to the released version.